solutions/usaco-640.mdx

```diff
@@ -218,4 +218,70 @@ public class BCS {
 
 ```
 </JavaSection>
+<PySection>
+```py
+from sys import exit
+
+with open('bcs.in') as read:
+    n, k = [int(i) for i in read.readline().split()]
```

Comment:
crap man can you also change the indentation to tabs

Reply:
No worries, I'll fix it